### PR TITLE
Add k8s cluster addons

### PR DESCRIPTION
**Pull request by paaanic**
CMPT-31466 CMPT-31465

### CODE REVIEW

File: templates/kubernetes/data-sources/vkcs_kubernetes_addon/main.tf [hunk 1]
```diff
@@ -0,0 +1,5 @@
+data "vkcs_kubernetes_addon" "kube-prometheus-stack" {
+  cluster_id = vkcs_kubernetes_cluster.k8s-cluster.id
+  name = "kube-prometheus-stack"
+  version = "36.2.0"
```
Comment (ftersin) [inline templates/kubernetes/data-sources/vkcs_kubernetes_addon/main.tf:4]: lets move to examples

File: templates/kubernetes/data-sources/vkcs_kubernetes_addons/main.tf [hunk 2]
```diff
@@ -0,0 +1,3 @@
+data "vkcs_kubernetes_addon" "kube-prometheus-stack" {
```
Comment (ftersin) [inline templates/kubernetes/data-sources/vkcs_kubernetes_addons/main.tf:1]: lets move to examples

File: examples/vkcs_kubernetes_addon/base.tf [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  name = "ext-net"
+}
+
+resource "vkcs_networking_router" "k8s" {
+  name                = "k8s-router"
+  admin_state_up      = true
+  external_network_id = data.vkcs_networking_network.extnet.id
+}
+
+resource "vkcs_networking_router_interface" "k8s" {
```
Comment (ftersin) [inline examples/vkcs_kubernetes_addon/base.tf:23]: could not we use reusing pattern? for all entities

File: examples/vkcs_kubernetes_addon/import.sh [hunk 4]
```diff
@@ -0,0 +1 @@
+terraform import vkcs_kubernetes_addon.addon 141a1f77-0e89-4b63-8d75-1b4ae496f862/a94c8ae2-0cac-4795-9253-d23ce2a70f86
```
[the remark this answers is not in the document]
Comment (paaanic) [reply examples/vkcs_kubernetes_addon/import.sh]: Moved.

File: templates/kubernetes/data-sources/vkcs_kubernetes_addon/main.tf (hunk 1 above)
Comment (paaanic) [reply templates/kubernetes/data-sources/vkcs_kubernetes_addon/main.tf]: Moved.

File: templates/kubernetes/data-sources/vkcs_kubernetes_addons/main.tf (hunk 2 above)
Comment (paaanic) [reply templates/kubernetes/data-sources/vkcs_kubernetes_addons/main.tf]: Moved.

File: examples/vkcs_kubernetes_addon/base.tf (hunk 3 above)
Comment (paaanic) [reply examples/vkcs_kubernetes_addon/base.tf]: Used base from node_group example.

**Pull request merged**